+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Tue Sep 12 17:28:22 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL lvm2-lockd
+ pkgs=lvm2-lockd
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs lvm2-lockd --node 1 --tempfile ''
Tue Sep 12 17:28:33 2023 - ERROR - pkgs:(lvm2-lockd) not found
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ systemctl start lvmlockd.service
Failed to start lvmlockd.service: Unit lvmlockd.service not found.
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Tue Sep 12 17:28:34 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Tue Sep 12 17:28:34 2023 - INFO  - Start testing...
+ test_execution lvmlocks.service
+ service=lvmlocks.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-09-12 17:28:34'
+ test_restart lvmlocks.service
+ service=lvmlocks.service
+ systemctl restart lvmlocks.service
Failed to restart lvmlocks.service: Unit lvmlocks.service not found.
+ CHECK_RESULT 5 0 0 'lvmlocks.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='lvmlocks.service restart failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'lvmlocks.service restart failed'
+ LOG_ERROR 'lvmlocks.service restart failed'
+ message='lvmlocks.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlocks.service restart failed'
Tue Sep 12 17:28:35 2023 - ERROR - lvmlocks.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Tue Sep 12 17:28:35 2023 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lvmlocks.service
+ grep 'Active: active'
Unit lvmlocks.service could not be found.
+ CHECK_RESULT 1 0 0 'lvmlocks.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lvmlocks.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lvmlocks.service restart failed'
+ LOG_ERROR 'lvmlocks.service restart failed'
+ message='lvmlocks.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlocks.service restart failed'
Tue Sep 12 17:28:41 2023 - ERROR - lvmlocks.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Tue Sep 12 17:28:42 2023 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop lvmlocks.service
Failed to stop lvmlocks.service: Unit lvmlocks.service not loaded.
+ CHECK_RESULT 5 0 0 'lvmlocks.service stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='lvmlocks.service stop failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'lvmlocks.service stop failed'
+ LOG_ERROR 'lvmlocks.service stop failed'
+ message='lvmlocks.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlocks.service stop failed'
Tue Sep 12 17:28:42 2023 - ERROR - lvmlocks.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Tue Sep 12 17:28:43 2023 - ERROR - ../common/common_lib.sh line 39
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lvmlocks.service
+ grep 'Active: inactive'
Unit lvmlocks.service could not be found.
+ CHECK_RESULT 1 0 0 'lvmlocks.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lvmlocks.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lvmlocks.service stop failed'
+ LOG_ERROR 'lvmlocks.service stop failed'
+ message='lvmlocks.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlocks.service stop failed'
Tue Sep 12 17:28:49 2023 - ERROR - lvmlocks.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Tue Sep 12 17:28:50 2023 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start lvmlocks.service
Failed to start lvmlocks.service: Unit lvmlocks.service not found.
+ CHECK_RESULT 5 0 0 'lvmlocks.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='lvmlocks.service start failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'lvmlocks.service start failed'
+ LOG_ERROR 'lvmlocks.service start failed'
+ message='lvmlocks.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlocks.service start failed'
Tue Sep 12 17:28:50 2023 - ERROR - lvmlocks.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Tue Sep 12 17:28:51 2023 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lvmlocks.service
+ grep 'Active: active'
Unit lvmlocks.service could not be found.
+ CHECK_RESULT 1 0 0 'lvmlocks.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lvmlocks.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lvmlocks.service start failed'
+ LOG_ERROR 'lvmlocks.service start failed'
+ message='lvmlocks.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lvmlocks.service start failed'
Tue Sep 12 17:28:57 2023 - ERROR - lvmlocks.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Tue Sep 12 17:28:57 2023 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled lvmlocks.service
+ service=lvmlocks.service
++ systemctl is-enabled lvmlocks.service
+ state=not-found
+ '[' not-found == enabled ']'
+ '[' not-found == disabled ']'
+ '[' not-found == masked ']'
+ '[' not-found == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Tue Sep 12 17:28:58 2023 - INFO  - Unit is indirect, ignoring.
+ journalctl --since '2023-09-12 17:28:34' -u lvmlocks.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of lvmlocks.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of lvmlocks.service'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_reload lvmlocks.service
+ service=lvmlocks.service
+ systemctl start lvmlocks.service
Failed to start lvmlocks.service: Unit lvmlocks.service not found.
+ systemctl reload lvmlocks.service
+ grep 'Job type reload is not applicable'
+ CHECK_RESULT 1 0 0 'Job type reload is not applicable for unit lvmlocks.service'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit lvmlocks.service'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Job type reload is not applicable for unit lvmlocks.service'
+ LOG_ERROR 'Job type reload is not applicable for unit lvmlocks.service'
+ message='Job type reload is not applicable for unit lvmlocks.service'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Job type reload is not applicable for unit lvmlocks.service'
Tue Sep 12 17:28:59 2023 - ERROR - Job type reload is not applicable for unit lvmlocks.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 91'
+ message='../common/common_lib.sh line 91'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 91'
Tue Sep 12 17:28:59 2023 - ERROR - ../common/common_lib.sh line 91
+ return 0
+ systemctl status lvmlocks.service
+ grep 'Active: active'
Unit lvmlocks.service could not be found.
+ systemctl status lvmlocks.service
+ grep 'inactive (dead)'
Unit lvmlocks.service could not be found.
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Tue Sep 12 17:29:00 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Sep 12 17:29:00 2023 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Tue Sep 12 17:29:01 2023 - INFO  - start environment cleanup.
+ systemctl stop lvmlockd.service lvmlocks.service
Failed to stop lvmlockd.service: Unit lvmlockd.service not loaded.
Failed to stop lvmlocks.service: Unit lvmlocks.service not loaded.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Tue Sep 12 17:29:01 2023 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Tue Sep 12 17:29:02 2023 - INFO  - Finish environment cleanup!
